### ISSUE TITLE
Viewing anything!!!!!!!!!!!!!!

### ISSUE THREAD

**Issue by Gregtheone** [bdsmlr-development/bdsmlr-updates#50]
On one of your updates, you say that the hosting problem has been fixed and we can post and view images. LIKE HELL YOU CAN!!!!!I've cleared my browsing history dumped the cashe and I still can't see a DAMN thing!!!What more do I have to do! It's been like this for 4 to 5 months. This isn't the first time I've posted about this.....

**Comment by musikele** [comment]
The only way for me to make it work for me is to click on Settings and enable Next and Previous Pages Dashboard (Disable Infinite Scroll)

**Comment by Gregtheone** [comment]
Yes, disable infinite scroll is on, also on for archives and public blogs.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by Scott16546** [comment]
I could get it working on m1 mac by reloading it million times but lately, I can't see anything
